_assets/sass/components/_pb.scss

```diff
@@ -102,12 +102,14 @@
   .highlight, .pb-code-hl {
     margin-top: 20px;
     margin-bottom: 20px;
-    padding: 20px 22px 30px;
+    padding: 20px 22px 20px;
     border: 1px solid $t-divider-color;
     background: $t-code-bg;
     font-size: 16px;
 
     pre {
+      margin-top: 0;
```

Comment:
This was getting an additional 20px on top and bottom due to inheritance from .highlight class.